src/js/wrapper.ts

```diff
@@ -39,5 +41,90 @@ export const NATIVE = {
     return RNSentry.sendEvent(event);
   },
 
+  /**
+   * Starts native with dsn string and options.
+   * @param dsn string
+   * @param options ReactNativeOptions
+   */
+  async startWithDsnString(
+    dsn: string,
+    options: ReactNativeOptions
+  ): Promise<boolean> {
+    if (this.isNativeClientAvailable()) {
+      // tslint:disable-next-line: no-unsafe-any
+      return RNSentry.startWithDsnString(dsn, options);
+    }
+    return Promise.reject();
```

Comment:
Async await sytax.
Also, throw an error with a description:

`throw new SentryError('Native Client is not available can'\t start on native')`